src/Telegram.Bot/Types/VideoChatEnded.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Telegram.Bot.Types;
+
+/// <summary>
+/// This object represents a service message about a video chat ended in the chat.
+/// </summary>
+[JsonObject(MemberSerialization.OptIn, NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
+public class VideoChatEnded
+{
+    /// <summary>
+    /// Voice chat duration; in seconds
+    /// </summary>
+    [JsonProperty(Required = Required.Always)]
```

Comment:
why `Duration` field is `Required = Required.Always` vs `Users[]` is `DefaultValueHandling = DefaultValueHandling.Ignore` in `VideoChatParticipantsInvited`?

Reply:
It's a but from previous versions, will fix it